### ISSUE TITLE
Readme missing

### ISSUE THREAD

**Issue by rugk** [webvariants/policat#16]
I saw #2, but that is not enough. Every repo should have a Readme describing the basics apsect of the project.

And as said in #2 it should also link to the installation guide.

https://opensource.guide/starting-a-project/#launching-your-own-open-source-project

**Comment by ddzieduch** [comment]
Instructions from [doc/](https://github.com/webvariants/policat/tree/master/doc) are pretty straightforward.

The only difficulty for me is environmental settings in `docker-compose.yml` like:

```user: "${USER_UID:-1000}:${USER_GID:-1000}"```

```
WWW_USER: "${WWW_USER:-martin}"
WWW_GROUP: "${WWW_GROUP:-martin}"
```

I have to change them to run the project.

Besides, moving to README.md doesn't make much difference to me.

**Comment by rugk** [comment]
No, not moving. You can **link** to the docs folder, no problem.
But every project should have a Readme describing the project, linking to moire resources etc.

It's as if you are missing the homepage on your website.

See also the links I've given.

**Comment by ldidry** [comment]
Indeed: because there is no README, my reflex was to look at the issues to find a way to install the software, not to look at folders, hoping to find something that would be useful. An `INSTALL` file would have catch my attention, but certainly not a `doc` folder (catching the attention is why `README`, `LICENSE`, `INSTALL`, `CONTRIBUTING`, etc. are uppercase).
